docs/src/guides/deployment/bundlr-cli.md

```diff
@@ -0,0 +1,50 @@
+# Bundlr CLI
+
+## Requirements
+An Arweave wallet is required to deploy. If the size of the directory is greater than 100kb, a <a href="#fund-bundlr">funded Bundlr instance is required</a>.
+
+## Installation
+
+To install arkb run 
+```console
+yarn global add @bundlr-network/client
+```
+or 
+```console
+npm install -g @bundlr-network/client
+```
+
+## Static Build
+Permaweb applications are statically generated, meaning that the code and content are generated ahead of time and stored on the network.
+
+Below is an example of a static site. To deploy this to the Permaweb, the `build` directory will be passed in as the argument for the `upload-dir` flag.
+
+```js
+|- build
+    |- index.html
+    |- styles.css
+    |- index.js
+```
+
+## Deploying
+
+```console
+bundlr upload-dir [path to folder] -w [path to wallet] -c [currency] -h [bundlr node]
```

Comment:
Do you need to reference the index-file if deploying a web-app

--index-file <string>        
Name of the file to use as an index for upload-dir manifests (relative to the path provided to upload-dir).

Reply:
good catch! Added.